doc/lua_api.md

```diff
@@ -7898,6 +7898,7 @@ It can be created via `Settings(filename)`.
     * Also, see documentation for set() above.
 * `remove(key)`: returns a boolean (`true` for success)
 * `get_names()`: returns `{key1,...}`
+* `has(key)`: returns a boolean indicating whether `key` exists
```

Comment:
Add a note here that the key can still exist in parent settings layers. So, for example, `has(key) == false` does not mean that the default should be used.